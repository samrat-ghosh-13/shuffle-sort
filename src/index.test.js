const fs = require("fs");
const path = require("path");
const html = fs.readFileSync(path.resolve(__dirname, "./index.html"), "utf-8");

jest.dontMock("fs");

describe("Testing", () => {
	beforeEach(() => {
		document.documentElement.innerHTML = html.toString();
	});
	afterEach(() => {
		jest.resetModules();
	});
	it("Match Snapshots", () => {
		expect(document.documentElement.innerHTML).toMatchSnapshot();
	});
	it("Shuffle Button", () => {
		expect(document.getElementsByName("Shuffle")).toBeTruthy();
	});
	it("Sort Button", () => {
		expect(document.getElementsByName("Shuffle")).toBeTruthy();
	});
});
