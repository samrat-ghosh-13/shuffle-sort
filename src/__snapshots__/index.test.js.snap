// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Match Snapshots 1`] = `
"<head>
		<meta charset="UTF-8">
		<meta http-equiv="content-language" content="en">
		<meta http-equiv="X-UA-Compatible" content="IE=edge">
		<meta name="viewport" content="width=device-width, initial-scale=1.0">
		<title>Shuffle and Sort by Samrat Ghosh</title>
		<meta name="description" content="Shuffle and Sort by Samrat Ghosh">
		<link rel="stylesheet" type="text/css" href="./styles/styles.css">
		<script type="text/javascript" src="./helpers/index.js" defer=""></script>
	</head>
	<body>
		<section id="root"></section>
	

</body>"
`;
